Tests/Clappr_Tests/AVFoundationPlaybackTests.swift

```diff
@@ -933,6 +933,207 @@ class AVFoundationPlaybackTests: QuickSpec {
                 }
             }
             
+            describe("#AVFoundationPlayback states") {
+                describe("#idle") {
+                    context("when ready to play") {
+                        it("current state must be idle") {
+                            let playback = AVFoundationPlayback()
+                            
+                            expect(playback.currentState).to(equal(.idle))
+                        }
+                    }
+
+                    context("when play is called") {
+                        it("changes isPlaying to true") {
+                            let options = [kSourceUrl: "http://localhost:8080/sample.m3u8"]
+                            let playback = AVFoundationPlayback(options: options)
+
+                            playback.play()
+
+                            expect(playback.isPlaying).to(beTrue())
+                            expect(playback.currentState).toEventually(equal(.playing), timeout: 3)
+                        }
+                    }
+
+                    context("when pause is called") {
+                        it("changes current state to paused") {
+                            let playback = AVFoundationPlayback()
+
+                            playback.pause()
+                            
+                            expect(playback.isPaused).to(beTrue())
```

Comment:
Same as above. `isPlaying` and `isBuffering` must be `false`, right?

Reply:
Done.